### PR TITLE
BCN - Nicolas Lacaze - WDFT0119

### PR DESCRIPTION
**Pull request by nicolacaze**

### CODE REVIEW

File: starter_code/app.js [hunk 1]
```diff
@@ -1,21 +1,41 @@
 
 const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
 const path    = require('path');
+const expressLayouts = require('express-ejs-layouts');
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 const punkAPI = new PunkAPIWrapper();
```
Comment (axelgar) [inline starter_code/app.js]: Try to separate the requires from other variables that you add. It's a way to keep your code more structure.

File: starter_code/public/stylesheets/styles.css [hunk 2]
```diff
@@ -5,4 +5,159 @@ dark-blue: #0A2C42;
 grey: rgba(145, 145, 145, 0.481);
 font-family: Arial, Helvetica, sans-serif
 */
+
```
Comment (axelgar) [inline starter_code/public/stylesheets/styles.css]: Good that you used the boilerplate, but if there are things you don't use, it's better if you delete them.

File: starter_code/app.js (hunk 1 above)
Comment (nicolacaze) [reply starter_code/app.js]: Noted!

File: starter_code/public/stylesheets/styles.css (hunk 2 above)
Comment (nicolacaze) [reply starter_code/public/stylesheets/styles.css]: Gotcha 🧐

**Pull request closed without merging**